src/main/java/io/jenkins/plugins/orka/OrkaAgent.java

```diff
@@ -46,12 +49,15 @@ public class OrkaAgent extends AbstractCloudSlave {
     private String namePrefix;
     private String jvmOptions;
 
+    private final List<PortMapping> portMapping;
```

Comment:
As this is a list, plural sounds better - `portMappings`

Reply:
Corrected